pkgs/by-name/eq/equicord/package.nix

```diff
@@ -9,13 +9,13 @@
 }:
 stdenv.mkDerivation (finalAttrs: {
   pname = "equicord";
-  version = "1.10.8"; # from package.json
+  version = "1.10.8";
 
   src = fetchFromGitHub {
     owner = "Equicord";
     repo = "Equicord";
-    rev = "935a5eaf6e5894294ec45ec540e9ecb07e850de0";
-    hash = "sha256-dfOzASBP0dEArJXuddLfPLZ7IcsEfKll+ju1jcHarNk=";
+    rev = "refs/tags/v${finalAttrs.version}";
```

Comment:
Use `tag = finalAttrs.version` here.

Reply:
As noted in the PR description, using `tag` breaks `env.EQUICORD_HASH = "${finalAttrs.src.tag}"`

I might be missing something, but building with tag will result in:

```bash
error: attribute 'tag' missing
       at /nix/store/00i5cbz0ww2grlv0q9rmyl4s62p95ay1-source/pkgs/by-name/eq/equicord/package.nix:35:24:
           34|     EQUICORD_REMOTE = "${finalAttrs.src.owner}/${finalAttrs.src.repo}";
           35|     EQUICORD_HASH = "${finalAttrs.src.tag}";
             |                        ^
           36|   };
```

Reply:
Please rebase, tag argument was not exposed previously. Should be fixed with https://github.com/hsjobeki/nixpkgs/commit/ce33e20a03de383dea7d04ec5680d342a8345493

Reply:
Done.